### DISCUSSION TITLE
Vertical space is prescious. Should Catalyst use sidebars instead?

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#472] [Polls]
Should common Catalyst functionality be moved to a sidebar?

**Comment by redstone-dev** [comment]
What constitutes "common Catalyst functionality?"

**Reply by jdev082** [reply]
Essentially moving the more menu options, bookmarks toggle, and bookmark page options into a compact sidebar. The preferences and bookmarks pages would likely get moved to sidebars as well.

**Reply by jdev082** [reply]
There `vertical-tabs-experiment` branch of Catalyst has an implementation of vertical tabs. Which would essentially be how this sidebar system would be implemented.
